src/main/groovy/info/solidsoft/gradle/pitest/PitestPlugin.groovy

```diff
@@ -59,6 +60,7 @@ class PitestPlugin implements Plugin<Project> {
             task.with {
                 description = "Run PIT analysis for java classes"
                 group = PITEST_TASK_GROUP
+                shouldRunAfter(project.tasks.named(TEST_TASK_NAME))
```

Comment:
This is only available in Gradle [4.10+](https://docs.gradle.org/current/javadoc/org/gradle/api/NamedDomainObjectCollection.html#named-java.lang.String-). I will bump the minimal required Gradle version to ~4.10 soon (in the aforementioned refactoring), so let's keep this PR open and I will merge it after it.

Reply:
Sooo :-)